<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralConfigPage</class>
 <widget class="QWidget" name="GeneralConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>On application start:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="startNewSessionOption">
     <property name="text">
      <string>Start a new session</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">startSessionOptions</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="startLastSessionOption">
     <property name="text">
      <string>Load last session</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">startSessionOptions</string>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="startNamedSessionOption">
       <property name="text">
        <string>Load session:</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">startSessionOptions</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="startSessionName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="useLanguageConfigFromSession">
     <property name="text">
      <string>Save different language configuration per session.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>FileTree Root:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ScIDE::PathChooserWidget" name="fileTreeRootDir"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>250</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScIDE::PathChooserWidget</class>
   <extends>QLineEdit</extends>
   <header>path_chooser_widget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="startSessionOptions"/>
 </buttongroups>
</ui>
